cpp-rt.hpp

```diff
@@ -0,0 +1,100 @@
+#ifndef CPPRT_HPP
+#define CPPRT_HPP
+
+#include <stddef.h>
+#include <malloc.h>
```

Comment:
since we have cstdlib, is this required?

Reply:
Nop, you are right